### ISSUE TITLE
[bug]: inactive chips have no visible focus

### ISSUE THREAD

**Issue by alizedebray** [swisspost/design-system#3753]
**Describe the bug**
The focus ring of inactive filter chips is not visible.

**To Reproduce**

1. Go to [the chips documentation](https://design-system.post.ch/?path=/docs/12576d97-52c3-49ec-be7b-6d37728b75f5--docs)
2. Focus on the first chips example
3. No focus ring is visible

**Expected behavior**
The focus ring should be visible.

**Desktop (please complete the following information):**

- OS: all
- Browser: all
- Version: all
